src/X.Serilog.Sinks.Telegram/Batch/LogsQueueAccessContext.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Concurrent;
+
+namespace X.Serilog.Sinks.Telegram.Batch;
+
+internal class LogsQueueAccessContext : ILogsQueueAccessor
+{
+    private readonly ConcurrentQueue<string> _logs = new();
+
+    public void Enqueue(string log)
+    {
+        _logs.Enqueue(log);
+    }
+
+    public Task<List<string>> DequeueSeveralAsync(int amount)
+    {
+        var logs = new List<string>();
```

Comment:
Not a big deal, but can pass in amount as argument of List constructor. As it seems to be performance optimized code.